packages/web-components/fast-components/src/text-field/text-field.ts

```diff
@@ -108,8 +107,6 @@ export class TextField extends FormAssociated<HTMLInputElement> {
         }
     }
 
-    @attr
-    public value: string;
```

Comment:
is the duplication of these from the inherited abstract class causing the issue?

Reply:
it was caused by defining a different converter for the inherited property, yes - multiple `@attr` on the property is a no-no.

I've cleaned them all up except for slider which converts the `value` prop to a number. We'll want to address that but I think it is a larger change we should isloate.

Reply:
gotcha - assuming the issue would persist if the `@attr` had a different default value here which may be something that needs to be addressed as well?

Reply:
nm pulling it off slider seems to work for the examples so pulled it off. There might be some bugs but I think we can harden it in a second pass.